en/identity-server/next/docs/get-started/try-samples/qsg-oidc-webapp-java-ee.md

```diff
@@ -195,6 +195,55 @@ Follow the steps given below to configure the sample app.
     sh catalina.sh start
     ```
 
+## Update the java keystore
+
+By default, tomcat is using the default Java keystore (cacerts) to build the SSL connection. In {{ product_name }}, the default certificate is a self signed
+certificate. This certificate needs to be added to the Java keystore. Please follow the given steps below to extract the public key from {{ product_name }} 
+keystore and import it to the Java keystore.
+
+1. Export the public key from {{ product_name }} keystore.
+
+    - Command
+    ``` shell
+    keytool -export -alias {{CERT_ALIAS}} -file {{CERT_NAME}} -keystore {{PATH_TO_KEYSTORE}} -storepass {{KEYSTORE_PASSWORD}}
+    ```
+
+    - Sample
+    ``` shell
+    keytool -export -alias wso2carbon -file carbon_public2.crt -keystore wso2carbon.p12 -storepass wso2carbon
```

Comment:
Chall we mention the default keystore location as well in IS. Also the default key store is .jks file.